.gitignore

```diff
@@ -42,3 +42,7 @@ target/
 *.scale
 justfile
 rustc-ice-*
+.bacon-locations
+bacon.log
+bacon-ls.log
+.config/bacon.toml
```

Comment:
I guess its fine to add it here. However, you can also just add files in system wide git ignore file on your computer. Otherwise we will be adding files for every editor under the sun here.

Reply:
Fair point! Reverted gitignore changes: [14d5d9a](https://github.com/paritytech/polkadot-sdk/pull/8118/commits/14d5d9a6e68fcd63c0cfa4765f06c553d4d4efc5).